wgpu-hal/src/noop/mod.rs

```diff
@@ -83,7 +109,31 @@ impl crate::Instance for Context {
         &self,
         _surface_hint: Option<&Context>,
     ) -> Vec<crate::ExposedAdapter<Api>> {
-        Vec::new()
+        vec![crate::ExposedAdapter {
+            adapter: Context,
+            info: wgt::AdapterInfo {
+                name: String::from("noop wgpu backend"),
+                vendor: 0,
+                device: 0,
+                device_type: wgt::DeviceType::Cpu,
+                driver: String::from("wgpu"),
+                driver_info: String::new(),
+                backend: wgt::Backend::Noop,
+            },
+            features: wgt::Features::all(),
+            capabilities: crate::Capabilities {
+                limits: wgt::Limits::default(),
```

Comment:
This should probably be spelled out as something high. Potentially just u32::MAX/1 depending on max/min limit type.